pkg/security/tests/module_tester_linux.go

```diff
@@ -632,6 +632,12 @@ func newTestModuleWithOnDemandProbes(t testing.TB, onDemandHooks []rules.OnDeman
 		opt(&opts)
 	}
 
+	prevEbpfLessEnabled = ebpfLessEnabled
```

Comment:
```suggestion
	prevEbpfLessEnabled := ebpfLessEnabled
```
I think this breaks the test build